scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala

```diff
@@ -46,6 +46,21 @@ class H2OMOJOModel(override val uid: String) extends H2OMOJOModelBase[H2OMOJOMod
 
   def getModelDetails(): String = $(modelDetails)
 
+  def getDomainValues(): Array[Array[String]] = {
```

Comment:
Do Python and R converters support nested collections? Shouldn't we cover it with tests?

Reply:
Good idea, will test

Reply:
Fixed Python & added test for Python & R